sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -180,6 +182,7 @@ export default defineComponent({
         isReadOnly: isReadOnly.value,
         isFulltimeAllowed,
       };
+      endDate.value = programYear.programYearEndDate;
```

Comment:
To let you know, programYearEndDate is not the study end date. Study end date refers to offering end date.
And for this particular ticket it is either offering end date or program info end date.